### ISSUE TITLE
Build Gradle Duplicate Files Error

### ISSUE THREAD

**Issue by DaveBben** [Grover-c13/PokeGOAPI-Java#342]
I have the development build (dcaa793557)  in my build gradle but I'm getting an error with lots of duplicate files (META - INF) from the moshi jar and Pokémon go jar. Anyone else?

**Comment by vmarchaud** [comment]
Can you give us the complete log of the build ? (via pastebin plz)

**Comment by vaibhavthapliyal** [comment]
You can try and add this to your build.gradle:

 packagingOptions {
        exclude 'META-INF/maven/com.squareup.moshi/moshi/pom.properties'
        exclude 'META-INF/maven/com.squareup.moshi/moshi/pom.xml'

```
    exclude 'META-INF/maven/com.google.protobuf/protobuf-java/pom.xml'
    exclude 'META-INF/maven/com.google.protobuf/protobuf-java/pom.properties'
    exclude 'META-INF/LICENSE.txt'
    exclude 'META-INF/NOTICE.txt'
    exclude 'META-INF/maven/com.squareup.okio/okio/pom.xml'
    exclude 'META-INF/maven/com.squareup.okio/okio/pom.properties'

    exclude 'META-INF/maven/com.squareup.okhttp3/okhttp/pom.properties'
    exclude 'META-INF/maven/com.squareup.okhttp3/okhttp/pom.xml'

    exclude 'META-INF/maven/com.annimon/stream/pom.properties'
    exclude 'META-INF/maven/com.annimon/stream/pom.xml'

    pickFirst 'gpsoauth.properties'
    pickFirst 'google/protobuf/any.proto'
    pickFirst 'google/protobuf/field_mask.proto'
    pickFirst 'google/protobuf/plugin.proto'
    pickFirst 'google/protobuf/compiler/plugin.proto'
    pickFirst 'google/protobuf/api.proto'
    pickFirst 'google/protobuf/descriptor.proto'
    pickFirst 'google/protobuf/wrappers.proto'
    pickFirst 'google/protobuf/type.proto'
    pickFirst 'google/protobuf/struct.proto'
    pickFirst 'google/protobuf/timestamp.proto'
    pickFirst 'google/protobuf/source_context.proto'
    pickFirst 'google/protobuf/duration.proto'
    pickFirst 'google/protobuf/empty.proto'



}
```

**Comment by DaveBben** [comment]
I added the packing options. Now I get the following error
Error:Execution failed for task ':app:transformClassesWithDexForDebug'.

> com.android.build.api.transform.TransformException: java.lang.RuntimeException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command 'E:\jdk\bin\java.exe'' finished with non-zero exit value 1

**Comment by vmarchaud** [comment]
@movieboy711 You should update the version of the lib, this issue has been fixed since (cf #326 )

Re-open if you have another problem.
